src/tests/tracing/runtimeeventsource/nativeruntimeeventsource.csproj

```diff
@@ -5,15 +5,9 @@
     <GCStressIncompatible>true</GCStressIncompatible>
     <!-- This test has a secondary thread with an infinite loop -->
     <UnloadabilityIncompatible>true</UnloadabilityIncompatible>
-    <!--
-      Test is blocking official build.
-      Related failures: #18907, #19340, #22441, #22729.
-      Issue tracking to re-enable: #22898.
-    -->
-    <DisableProjectBuild>true</DisableProjectBuild>
```

Comment:
None of these issue numbers seem related to this code. I'm going to poke at the old coreclr repo issues to see if these refer to those numbers (the transfer did some funny business with mapping the numbers).

Reply:
https://github.com/dotnet/runtime/issues/12141

This test has been off for >3 years now.  I want this test back on, but perhaps not as part of a changeset that is going to be backported. Thoughts?